qaframework-bdd-tests/cypress/support/commands.js

```diff
@@ -333,7 +333,7 @@ Cypress.Commands.add('createPatient', (identifier = null) => {
             {
                 identifier,
                 identifierType: '05a29f94-c0ed-11e2-94be-8c13b969e334',
-                location: '6351fcf4-e311-4a19-90f9-35667d99a8af',
+                location: 'ba685651-ed3b-4e63-9b35-78893060758a',
```

Comment:
```suggestion
                location: DEFAULT_LOCATION_UUID,
```
We can get the location the default location uuid global variable